tailwind.config.js

```diff
@@ -6,6 +6,9 @@ export default {
       'lato': ['Lato'],
       'oswald': ['Oswald'],
     },
+    fontSize: {
```

Comment:
This has to be moved under `extend`, otherwise you're making it impossible to use the other text sizes that Tailwind has built in. I changed this in the main branch, so you can just merge main into this branch to fix this.